partiql-spi/src/test/kotlin/org/partiql/value/TimestampTest.kt

```diff
@@ -1,928 +1,695 @@
-//             """
-//                 Expected PartiQL Timestamp : ${tc.expectedTimestamp},
-//                 Actual PartiQL Timestamp   : $actualTimestamp
-//             """.trimIndent()
-//         }
-//         assert(tc.expectedIonEquivalent == actualTimestamp.ionTimestampValue) {
-//             """
-//                 Expected Ion Timestamp : ${tc.expectedIonEquivalent},
-//                 Actual Ion Timestamp   : ${actualTimestamp.ionTimestampValue}
-//             """.trimIndent()
-//         }
-//         assert(tc.expectedIonEquivalent.decimalMillis == actualTimestamp.epochMillis) {
-//             """
-//                 Expected epoch in Millis : ${tc.expectedIonEquivalent.decimalMillis},
-//                 Actual epoch in Millis   : ${actualTimestamp.epochMillis}
-//             """.trimIndent()
-//         }
-//         val timestampFromIon = DateTimeValue.timestamp(tc.expectedIonEquivalent)
-//         // ion Timestamp all arbitrary position
-//         val expectedTimestampInArbitraryPrecision = tc.expectedTimestamp
-//         assert(timestampFromIon == expectedTimestampInArbitraryPrecision) {
-//             """
-//                 Expected Timestamp From Ion : $expectedTimestampInArbitraryPrecision,
-//                 Actual Timestamp From Ion : $timestampFromIon
-//             """.trimIndent()
-//         }
-//     }
+    private fun testFailedTestCases(tc: FailedTest) {
+        assertThrows<Exception> {
+            tc.statement()
+        }
+    }
 
-//     private fun testEqualsAndCompareToTest(tc: EqualsAndCompareToTest) {
-//         // Equals
-//         assert((tc.timestamp1 == tc.timestamp2) == tc.expectedEqualsRes) {
-//             """
-//                 Timestamp 1 : ${tc.timestamp1},
-//                 Timestamp 2 : ${tc.timestamp2},
-//                 Expected Equals method result : ${tc.expectedEqualsRes}
-//                 Actual Equals method result : ${tc.timestamp1 == tc.timestamp2}
-//             """.trimIndent()
-//         }
-//         // Compare To:
-//         assert((tc.timestamp1.compareTo(tc.timestamp2)) == tc.expectedCompareToRes) {
-//             """
-//                 Timestamp 1 : ${tc.timestamp1},
-//                 Timestamp 2 : ${tc.timestamp2},
-//                 Expected Equals method result : ${when (tc.expectedCompareToRes) {0 -> "Equals" 1 -> "More" else -> "Less"}}
-//                 Actual Equals method result : ${when (tc.timestamp1.compareTo(tc.timestamp2)) {0 -> "Equals" 1 -> "More" else -> "Less"}}
-//             """.trimIndent()
-//         }
-//     }
+    companion object {
+        private const val LESS = -1
+        private const val EQUALS = 0
+        private const val MORE = 1
 
-//     private fun testFailedTestCases(tc: FailedTest) {
-//         assertThrows<DateTimeException> {
-//             tc.statement()
-//         }
-//     }
+        private val successCaseWithKnownTimeZone = listOf(
```

Comment:
It's slightly tricky for me to compare the diffs of the tests from this point onwards. Are there any test cases that needed modified or were deleted? The migrated version is using about 200 less lines of code.

Reply:
I see. Thanks for the clarification.